source/_components/alarm_control_panel.ifttt.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: "IFTTT Alarm Control Panel"
+description: "Instructions how to integrate IFTTT-controlled security systems into Home Assistant."
+date: 2018-03-10 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ifttt.png
+ha_category: Alarm
+ha_release: ?
+---
+
+The `ifttt` platform allows you to integrate security systems that have no open API, but can be controlled through [IFTTT](https://ifttt.com/discover).
+
+This platform depends on the [IFTTT](https://home-assistant.io/components/ifttt/) Home Assistant component. See the component's documentation to set it up.
+
+<p class='note'>
+It is important to note that this platform fully relies on IFTTT to receive updates when the security system's state changes. Therefore, this platform shows an assumed state.
+</p>
+
+To enable this, setup the required IFTTT applets as listed below and add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+ifttt:
+  key: YOUR_WEBHOOK_KEY
+  
+alarm_control_panel:
+  - platform: ifttt
+    name: YOUR_ALARM_NAME
+    code: YOUR_ALARM_CODE
+```
+
+<p class='note warning'>
+It is strongly discouraged to use this platform when you don't use encryption, otherwise your API password will be send unprotected through the IFTTT Webhooks. It is adviced to [setup encryption using Let's Encrypt](https://home-assistant.io/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/).
+</p>
+
+{% linkable_title Required IFTTT applets %}
+
+This platform supports the services `alarm_disarm`, `alarm_arm_away`, `alarm_arm_home` and `alarm_arm_night`. For each of these services, an IFTTT webhook will be triggered. 
+
+For this system to operate correctly, the following IFTTT applets have to be setup:
```

Comment:
Please note that not all alarm systems support all the modes. For example, my Abode security system does not have `alarm_arm_night`. It may be good to clarify that users should check the modes supported by their security system and add only the ones supported by their systems.